### ISSUE TITLE
Plug-ins s60v3

### ISSUE THREAD

**Issue by yfaizin1** [mrRosset/Symbian-Archive#38]
File plug-ins s60v3 fp1 and fp2 file same? size same

**Comment by xplt** [comment]
> File plug-ins s60v3 fp1 and fp2 file same? size same

Yes. I reported it here: https://github.com/mrRosset/Symbian-Archive/issues/30
